backend.native/tests/build.gradle

```diff
@@ -2415,6 +2419,12 @@ task interop_globals(type: RunInteropKonanTest) {
     interop = 'cglobals'
 }
 
+task interop_macros(type: RunInteropKonanTest) {
+    disabled = (project.testTarget == 'wasm32') // No interop for wasm yet.
```

Comment:
Even when it will be available, it will be JS and unlikely similar to what we have  for C. Maybe just // C interop is irrelevant to WASM.